addons/viewports/CfgVehicles.hpp

```diff
@@ -0,0 +1,169 @@
+class CfgVehicles {
+    class B_MBT_01_base_F;
+    class B_MBT_01_cannon_F: B_MBT_01_base_F { // Merkel
+        class ADDON {
```

Comment:
should ADDON macro really be used here?
As I understand it these macros are used so they auto update if you rename a component.
But if for some reason ace_viewports gets renamed in the future, we don't want this classname to change and break all mods that built upon this?

Reply:
It's just to make things easier, this won't ever realistically get renamed.